src/routes/pig.py

```
@@ -69,6 +70,7 @@ async def delete_pig(id: int, session: SessionDep, request: Request) -> None:
     pig = session.get(PIG, id)
     if not pig: raise HTTPException(404, detail="해당 id의 시그/피그가 없습니다")
     pig.status = SCSCStatus.inactive
+    if pig.status == SCSCStatus.inactive: raise HTTPException(400, detail="해당 시그/피그는 이미 비활성 상태입니다")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)